### ISSUE TITLE
inline equations overlap html text

### ISSUE THREAD

**Issue by ihojmanb** [webyonet/react-native-mathjax-html-to-svg#13]
Hi, I'm having issues rendering inline equations. Sometimes it works fine, but othertimes it overlaps with html text making it unintelligible.
![image](https://github.com/webyonet/react-native-mathjax-html-to-svg/assets/30912229/4577995f-6f9d-463f-b5f1-b9abc9d9c06f)

any guidance of how can I solve this? any workarounds?

**Comment by herni-dev** [comment]
Did you found a fix?

**Comment by ihojmanb** [comment]
not really, but I'm working on it. I will update this thread if something pops up ;)

**Comment by webyonet** [comment]
I would try if you shared html and latex

**Comment by ihojmanb** [comment]
@webyonet hi! I've written an issue [here](https://github.com/meliorence/react-native-render-html/issues/637) related to this since I'm using your library along `react-native-render-html`, but to put things straight, here is my minimal reproducible example: [https://github.com/ihojmanb/math-equations-app](https://github.com/ihojmanb/math-equations-app). 

There you will find the html, latex, and the way I'm using the libraries together. I hope that can help clarify the issue.

**Comment by herni-dev** [comment]
Hey @ihojmanb thanks for creating that issue on react native render html. I suspect that the problem is from that side. Hopefully we can fix this

**Comment by raysonance** [comment]
> @webyonet hi! I've written an issue [here](https://github.com/meliorence/react-native-render-html/issues/637) related to this since I'm using your library along `react-native-render-html`, but to put things straight, here is my minimal reproducible example: https://github.com/ihojmanb/math-equations-app.
> 
> There you will find the html, latex, and the way I'm using the libraries together. I hope that can help clarify the issue.

hey bro thanks for the rendering engine. really helped a lot with spacing. btw is that the finished rendering component?
